app/presenters/organisations/header_presenter.rb

```diff
@@ -76,7 +76,7 @@ def translation_links
         {
           brand: org.brand,
           no_margin_top: true,
-          translations: links
+          translations: links.reverse
```

Comment:
This assumes the order of translations in the content item - not sure if we can rely on that. Also think it makes sense to be consistent with how we sort translations in other places, e.g: in government-frontend we sort by locale.

`translations.sort_by { |t| t["locale"] == I18n.default_locale.to_s ? '' : t["locale"] }`